includes/api/class-wc-admin-rest-reports-customers-controller.php

```diff
@@ -39,25 +39,30 @@ class WC_Admin_REST_Reports_Customers_Controller extends WC_REST_Reports_Control
 	 */
 	protected function prepare_reports_query( $request ) {
 		$args                            = array();
-		$args['before']                  = $request['before'];
-		$args['after']                   = $request['after'];
+		$args['registered_before']       = $request['registered_before'];
+		$args['registered_after']        = $request['registered_after'];
 		$args['page']                    = $request['page'];
 		$args['per_page']                = $request['per_page'];
+		$args['order']                   = $request['order'];
+		$args['orderby']                 = $request['orderby'];
+		$args['match']                   = $request['match'];
 		$args['name']                    = $request['name'];
 		$args['username']                = $request['username'];
 		$args['email']                   = $request['email'];
 		$args['country']                 = $request['country'];
 		$args['last_active_before']      = $request['last_active_before'];
 		$args['last_active_after']       = $request['last_active_after'];
-		$args['order_count_min']         = $request['order_count_min'];
-		$args['order_count_max']         = $request['order_count_max'];
-		$args['order_count_between']     = $request['order_count_between'];
+		$args['orders_count_min']        = $request['orders_count_min'];
+		$args['orders_count_max']        = $request['orders_count_max'];
+		$args['orders_count_between']    = $request['orders_count_between'];
```

Comment:
Do we want to normalize this to set `orders_count_min` and `orders_count_max` if it's present?

Reply:
Definitely - this was missed and is why the `_between` params didn't work in your testing.

Reply:
On second thought.. perhaps we should remove that parameter and handle this normalization on the JS side?

Reply:
Nevermind on the JS-side normalization, I think that's perhaps the wrong place to perform this operation.

In an effort to land this PR, and to properly handle `_between` params for all reports that use the advanced number filter, I've opened https://github.com/woocommerce/wc-admin/issues/1239.

For this PR, the `_between` param will continue to be inert.

Reply:
Seems like a good way of handling this 👍